runtime/docker/network.go

```diff
@@ -54,7 +55,18 @@ func (c *client) InspectNetwork(ctx context.Context, b *pipeline.Build) ([]byte,
 		return nil, err
 	}
 
-	return []byte(n.ID + "\n"), nil
+	// convert network type NetworkResource to bytes with pretty print
+	//
+	// https://godoc.org/github.com/docker/docker/api/types#NetworkResource
+	network, err := json.MarshalIndent(n, "", " ")
+	if err != nil {
+		return nil, err
+	}
+
+	// add new line to end of bytes
+	network = append(network, "\n"...)
+
+	return network, nil
```

Comment:
I think we could condense these lines into a single line 👍 

Thoughts on that?

```suggestion
	// add new line to end of bytes
	return append(network, "\n"...), nil
```